app/helpers/form_errors_builder.rb

```diff
@@ -0,0 +1,29 @@
+class FormErrorsBuilder < ActionView::Helpers::FormBuilder
```

Comment:
As much as I like encapsulation and OO design, Nik's suggestion seems much more readable to someone unfamiliar with the codebase